docs/source/api_user.rst

```diff
@@ -9,7 +9,6 @@ These are reference class and function definitions likely to be useful to everyo
    intake.register_driver
    intake.unregister_driver
    intake.upload
-   intake.open_
```

Comment:
The idea was, that for an example driver like CSVSource, which has name="csv", a function will be generated intake.open_csv - and also for all other named and found drivers. This item was supposed to document this behaviour, but there is of course no function just called open_. Perhaps the API docs is the wrong place to put it and you are right to simply remove it here.

Reply:
Sphinx might have had an update or something recently, but it was very unhappy about `intake.open_` not being importable. I do think there are better places for it, but I wish Sphinx could understand this better.